- fix a unit test failing on OpenBSD/aarch64
  fa50380be027b66f702efb4bc5a37b1d2c35f4f2

Index: rts/System/creg/SerializeLuaState.cpp
--- rts/System/creg/SerializeLuaState.cpp.orig
+++ rts/System/creg/SerializeLuaState.cpp
@@ -1042,7 +1042,7 @@ void creg_lua_State::PostLoad()
 	}
 
 	size_t savedpc_offset = * (size_t *) &savedpc;
-	savedpc = GetProtoFromCallInfo(ci)->code + savedpc_offset;
+	savedpc = GetProtoFromCallInfo(ci - 1)->code + savedpc_offset;
 }
 
 
